src/daal4py.h

```diff
@@ -53,6 +53,10 @@ using daal::services::LibraryVersionInfo;
 #include "data_management/data/internal/finiteness_checker.h"
 #endif
 
+#if INTEL_DAAL_VERSION >= 20200002
```

Comment:
We can avoid failings with DPCPP if we will pay attention for beta (2021...) versions too.

Reply:
I added more complex and working version getting. It should work for both beta and mnt.